File: swift/Sources/sdnx/Types/MixSchema.swift
import Foundation

public struct MixSchema: SchemaValue {
    public let type: String = "mix"
    public var description: String?
    public var validators: [String: ValidatorInfo]?
    public var inner: [Schema]
    
    public init(inner: [Schema], description: String? = nil, validators: [String: ValidatorInfo]? = nil) {
        self.inner = inner
        self.description = description
        self.validators = validators
    }
}

File: dotnet/Sdnx.Core/Types/MixSchema.cs
namespace Sdnx.Core
{
    public record MixSchema : SchemaValue
    {
        public required string Type { get; init; } = "mix";
        public required List<Dictionary<string, SchemaValue>> Inner { get; init; }
        public string? Description { get; init; }
        public Dictionary<string, Validator>? Validators { get; init; }
    }
}
